kong.conf.default

```diff
@@ -1532,6 +1532,96 @@
                                  # same name/type will be synchronized to a
                                  # single query.
 
+#------------------------------------------------------------------------------
+# New DNS RESOLVER
+#------------------------------------------------------------------------------
+
+# This DNS resolver introduces global caching for DNS records across workers,
+# significantly reducing the query load on DNS servers.
+#
+# It provides observable statistics, you can retrieve them through the Admin API
+# `/status/dns`.
+
+#legacy_dns_client = off         # Disable the new DNS resolver, using the
+                                 # original DNS resolver. See above `dns_xxx`
+                                 # options for the original DNS resolver.
+
+#resolver_address =              # Comma-separated list of nameservers, each
+                                 # entry in `ip[:port]` format to be used by
+                                 # Kong. If not specified, the nameservers in
+                                 # the local `resolv.conf` file will be used.
+                                 # Port defaults to 53 if omitted. Accepts
+                                 # both IPv4 and IPv6 addresses.
+
+#resolver_hostsfile = /etc/hosts # The hosts file to use. This file is read
```

Comment:
```suggestion
#resolver_hosts_file = /etc/hosts # The hosts file to use. This file is read
```

Reply:
Done (there are still other places that need to be modified.)